app/src/main/java/org/p2p/wallet/user/local/TokenListResponse.kt

```diff
@@ -1,42 +0,0 @@
-package org.p2p.wallet.user.local
-
-import com.google.gson.annotations.SerializedName
-
-data class TokenListResponse(
-
-    @SerializedName("tokens")
-    val tokens: List<TokenResponse>
```

Comment:
Why this class is deleted? 
It's being used while getting token metadata

Reply:
yeah, we have copy of this class in token-service module